tools/hangar/__snapshots__/test_corpus/sdk_tests/util/spawn.test.w_test_sim.md

```diff
@@ -0,0 +1,30 @@
+# [spawn.test.w](../../../../../../examples/tests/sdk_tests/util/spawn.test.w) | test | sim
+
+## stdout.log
+```log
+[INFO] spawn() with successful execution | Spawned child process with PID: 8718
```

Comment:
please either delete those logs or change the PID to a placeholder `Spawned child process with PID: <PID>`
[here](https://github.com/winglang/wing/blob/main/tools/hangar/src/utils.ts#L55)